### PR TITLE
more css fixes

### PR DESCRIPTION
**Pull request by cassiemak**
- Clear the data when closePin
- z-index works for user des page
- like button changed to red when liked
- change no result notifications

### CODE REVIEW

File: app/assets/stylesheets/user-account.css.scss [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
+  margin-top: 55px;
+}
+
 .profile-pix{
   width: 70px;
   height: auto;
   padding-top: 10px;
-  z-index: 2000;
+  z-index: 150;
+  position: relative;
```
Comment (paulld) [inline app/assets/stylesheets/user-account.css.scss:12]: So, that's what makes it work?

File: app/assets/templates/pin-show.html [hunk 2]
```diff
@@ -17,7 +17,7 @@
           
         <div class="image-content">
           <a href="{{ pin.source_url }}">
-            <img ng-src='{{ pin.img_sm_url }}' class="big-image">
+            <img src='{{ pin.img_sm_url }}' class="big-image">
```
Comment (paulld) [inline app/assets/templates/pin-show.html:20]: Why remove `ng-` ? cf. [ngSrc](https://docs.angularjs.org/api/ng/directive/ngSrc)

File: app/assets/stylesheets/user-account.css.scss (hunk 1 above)
Comment (cassiemak) [inline app/assets/stylesheets/user-account.css.scss:12]: Positive relative made it work

File: app/assets/templates/pin-show.html (hunk 2 above)
Comment (cassiemak) [inline app/assets/templates/pin-show.html:20]: Ng breaks the conditional function for no scrolling function when modal
pops up

Comment (paulld) [inline app/assets/templates/pin-show.html:20]: please remind me to have a look at that later!

**Pull request merged**